models/kernels/schaake_test.cpp

```diff
@@ -1,134 +0,0 @@
-#include "utility.hpp"
```

Comment:
I recommend making this like the other test file that incorporates the google test framework.